mf-dev/console/console-sidebar-config.json

```diff
@@ -33,5 +33,7 @@
     "url" : "",
     "clientId": "",
 	"enabled": false
-  }
+  },
+  "logo": "/de-logo.svg",
+  "mini-logo": "/de-mini.svg"
```

Comment:
Before I wrote you in chat...images files names are these:  "mini-logo.svg" and "DigitalEnabler.svg" for Console